XML/TestCases/FunctionalTests-Network.xml

```diff
@@ -123,7 +123,7 @@
         <Category>Functional</Category>
         <Area>NETWORK</Area>
         <Tags>network</Tags>
-        <Priority>2</Priority>
+        <Priority>3</Priority>
     </test>
     <test>
         <testName>NET-MAX-NICS</testName>
```

Comment:
Can we assign priority 0 for this test?

Reply:
Based on Azure update, it should be 1 not 0.

Reply:
Please check the entire xml to see the right priority:
- net max LEGACY NICs - this one has been moved as P3, we're talking about the actual hyper-v legacy networks.
- net max NICs - this one is already and remains as P1.

@juhlee-microsoft please dismiss Shital's review and merge this if @iamshital is off for the day.